dist/src/main/config/application.yaml

```diff
@@ -87,3 +87,30 @@ zeebe:
       # Controls the number of io threads to be used.
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_THREADS_IOTHREADCOUNT
       ioThreadCount: 2
+
+# Operate configuration properties
+
+camunda.operate:
+  # Set operate username and password.
+  # If user with <username> does not exists it will be created.
+  # Default: demo/demo
+  #username:
+  #password:
+  # ELS instance to store Operate data
+  elasticsearch:
+    # Cluster name
+    clusterName: elasticsearch
+    # URL
+    url: http://localhost:9200
+  # Zeebe instance
+  zeebe:
+    # Gateway address
+    gatewayAddress: localhost:26500
```

Comment:
we can reuse the value set in Zeebe
`gatewayAddress: localhost:${zeebe.broker.gateway.network.port:26500}`